domain/snyk/issues.go

```diff
@@ -363,18 +273,21 @@ func (i *Issue) UnmarshalJSON(data []byte) error {
 			return err
 		}
 		i.AdditionalData = codeData
+		break
```

Comment:
Why? In Go, afaik, you never need a break after a case - the default is break, you need to specify `fallthrough` if you need a fallthrough.